### ISSUE TITLE
keras 이용 classify 만들기

### ISSUE THREAD

**Issue by dizwe** [b-chae/VibrationFromWarningForHearingImpaired#12]
- Using keras template
   * DNN(V), CNN, RNN trails
- shorten the learning file length 
   * after short learning filter the wierd file
- amplitude based Learning V
- melSepctrogram based learning
- ensemble learning

ref?)
https://github.com/tensorflow/models/tree/master/research/audioset/yamnet
https://www.kaggle.com/c/freesound-audio-tagging/notebooks

**Comment by dizwe** [comment]
- Keras Template 사용하기 위해 sample code 돌아가도록 수정
    코드 수정이 조금 필요했음
    comet으로 ml 하이퍼파라미터와 결과 확인할 수 있음
- dataLoader 까지는 간단히 작성 완료.
    
Todo. np.stack, np.concatenate, append 비교해서 생각해볼것!
Todo.generator 안쓰고 넣으면 데이터가 너무 크지는 않을지 생각해보기

**Comment by dizwe** [comment]
학습 완료
[0.265428379819724, 0.9130781292915344] (loss, precision) 

Todo. classification report를 통해 각 class별 판별 현황 파악하기
```
              precision    recall  f1-score   support

           0       0.83      0.87      0.85       618
           1       0.84      0.88      0.86       693
           2       0.96      0.93      0.94      2451

    accuracy                           0.91      3762
   macro avg       0.88      0.89      0.89      3762
weighted avg       0.91      0.91      0.91      3762

```
Todo. 필요하면 다른 모델로 바꿔보기
Todo. np.stack, np.concatenate, append 비교해서 생각해볼것!
